<launch>
  <arg name="device" default="/dev/video0" />

  <node pkg="uvc_aggregator" type="uvc_aggregator_node" name="uvc_aggregator" output="screen">
    <remap from="image_raw" to="camera/rgb/image_color" />
    <remap from="camera_info" to="camera/rgb/camera_info" />
    <param name="device" value="$(arg device)" />
    <param name="width" type="int" value="320" />
    <param name="height" type="int" value="240" />
    <param name="fps" type="int" value="20" />
    <param name="frame" type="string" value="camera_link" />
    <!-- other supported params: auto_exposure, exposure_absolute, brightness, power_line_frequency -->
  </node>
</launch>
